target/linux/generic/files/drivers/net/phy/rtl8366_smi.h

```diff
@@ -64,6 +64,7 @@ struct rtl8366_smi {
 	u8			dbg_vlan_4k_page;
 #endif
 	u32			phy_id;
+	u8			chip;
```

Comment:
Why don't you use a enum for this? I think that would make the code much more readable.

Reply:
I was thinking about it, but it would require the definition of this enum in rtl8366_smi.h, if you think it would be better, I can do it.

Reply:
For me it gets confusing because you are not using normal numbers, but also shift this variable. 

I think it is no problem to define a enum in this file.

Reply:
Ok, I will do it in the evening, I will also change the chip to rtl8367b_chip to make it clear what it concerns.

Reply:
Done